pkg/cmd/command.go

```diff
@@ -18,13 +21,26 @@ func NewMigratorCommand() *cobra.Command {
 // NewMigrateCmd migrate database
 func NewMigrateCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "velamg",
+		Use:   "migrate",
 		Short: "migrate the database",
 		Long:  "migrate the database",
-		RunE: func(cmd *cobra.Command, args []string) error {
-			Migrate(cmd.Context())
-			return nil
-		},
+	}
+	cmd.RunE = func(cmd *cobra.Command, args []string) error {
+		if len(args) > 1 {
+			return fmt.Errorf("too many arguments")
+		}
+		if len(args) == 0 {
+			return fmt.Errorf("provide the path of the config file")
+		}
```

Comment:
It would be better to use `-c config.yaml` style instead of position argument.

Reply:
Yeah. It's better to use the -c flag.

Reply:
Done!!